internal/boxcli/shellenv.go

```diff
@@ -74,7 +77,60 @@ func shellEnvFunc(cmd *cobra.Command, flags shellEnvCmdFlags) (string, error) {
 		}
 	}
 
-	envStr, err := box.PrintEnv(cmd.Context(), flags.runInitHook)
+	opts := &devopt.PrintEnv{
+		Ctx:          cmd.Context(),
+		IncludeHooks: flags.runInitHook,
+	}
+	envStr, err := box.PrintEnv(opts)
+	if err != nil {
+		return "", err
+	}
+
+	return envStr, nil
+}
+
+type shellEnvOnlyPathWithoutWrappersCmdFlags struct {
+	config configFlags
+}
+
+func shellEnvOnlyPathWithoutWrappersCmd() *cobra.Command {
+	flags := shellEnvOnlyPathWithoutWrappersCmdFlags{}
+	command := &cobra.Command{
+		Use:     "only-path-without-wrappers",
```

Comment:
Nit, drop the only and let the command be "path-without-wrappers"

Reply:
🤔 I'd prefer leaving it in to make it clear that this is not also returning other env-vars of shellenv.